pages/linux/sar.md

```diff
@@ -0,0 +1,27 @@
+# sar
+
+> Monitor performance of various Linux subsystems.
+
+- Report I/O and Transfer rate every 1 second:
```

Comment:
What transfer rate are we talking about ?

Reply:
physical devices, added to the description